namespace java thrift.test

struct myBoolStruct {
  1: bool a
}
